controllers/storagecluster/cephcluster.go

```diff
@@ -1148,16 +1148,72 @@ func createPrometheusRules(r *StorageClusterReconciler, sc *ocsv1.StorageCluster
 		return err
 	}
 	applyLabels(getCephClusterMonitoringLabels(*sc), &prometheusRule.ObjectMeta)
-	replaceTokens := []exprReplaceToken{
+
+	replaceTokens := []replaceToken{
 		{
 			recordOrAlertName: "CephMgrIsAbsent",
-			wordInExpr:        "openshift-storage",
+			wordToReplace:     "openshift-storage",
 			replaceWith:       sc.Namespace,
 		},
 	}
+
+	// if different than default nearFullRatio or fullRatio values are provided, replace the values in the prometheus rule accordingly
+	if sc.Spec.ManagedResources.CephCluster.NearFullRatio != nil || sc.Spec.ManagedResources.CephCluster.FullRatio != nil {
+		//
+		var (
+			nearFullRatio = 0.75
+			fullRatio     = 0.85
```

Comment:
Define these values in the API package or default package.

Reply:
Done